sklearn/mixture/_gaussian_mixture.py

```diff
@@ -314,20 +314,27 @@ def _compute_precision_cholesky(covariances, covariance_type):
         precisions_chol = np.empty((n_components, n_features, n_features))
         for k, covariance in enumerate(covariances):
             try:
-                cov_chol = linalg.cholesky(covariance, lower=True)
+                cov_chol = linalg.cholesky(covariance,
+                                           lower=True,
+                                           check_finite=False)
```

Comment:
Can we assume that covariance will never be non finite ? We have `nk = resp.sum(axis=0) + 10 * np.finfo(resp.dtype).eps` line 278 and `covariances[k] = np.dot(resp[:, k] * diff.T, diff) / nk[k]` line 166.

On my machine I have 
```
>>> 1e300 / (10 * np.finfo(float).eps)
inf
```

So I would think we cannot assume that covariance will never be non finite ?

Reply:
You are right and I was thinking about it (see the message commit): in the case of a component being empty, this case might happen.

I also think that it is safer to remove this patch, I guess we need a maintainer's decision here.

Reply:
I agree to to revert this.